Sources/ProjectSpec/Scheme.swift

```diff
@@ -114,9 +114,11 @@ public struct Scheme: Equatable {
         public var launchAutomaticallySubstyle: String?
         public var debugEnabled: Bool
         public var simulateLocation: SimulateLocation?
+        public var executableName: String?
```

Comment:
I think we should remove the property from the `Build` struct as well.

Reply:
✅

Reply:
It looks like it's still there up on line 76?

Reply:
How about now?

Reply:
Perfect

Reply:
@brentleyjones  @Elland  When running tests on the CI I get the following error when I used `xcodegen` build from te latest `master` branch.

```
xcodebuild: error: The scheme 'MyTests' is not configured for Running.
--
08-Jun-2020 16:53:55 | The scheme 'MyTests' has nothing configured to build for Running and has no executable specified to Run. Edit the scheme to configure the Run action.
```

I believe the issue is caused by this change, because the previous release of xcodegen (0.15.1) works well for me.

I don't have an idea how to fix that. Do you now how to have the previous functionality working and changes that @Elland introduced?